requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+black
```

Comment:
I am not sure we want to add this file. Dev requirements are already specified through `setup.py` here https://github.com/MasoniteFramework/masonite/blob/4.0/setup.py#L188

You can install those by doing: `pip install 'masonite[test]'` 👉 that's what the makefile is doing when doing `make init` in a fresh development setup 😉.